rpc/query_server.go

```diff
@@ -1054,6 +1054,29 @@ func (s *QueryServer) EthGetBalance(address eth.Data, block eth.BlockHeight) (et
 	return eth.EncBigInt(*amount.Int), nil
 }
 
+// https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getStorageAt
+func (s *QueryServer) EthGetStorageAt(local eth.Data, position eth.Data, block eth.BlockHeight) (eth.Data, error) {
+	address, err := eth.DecDataToAddress(s.ChainID, local)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to decode address parameter %v", local)
+	}
+
+	positionBytes, err := eth.DecDataToBytes(position)
```

Comment:
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getstorageat is a confusing mess, according to the spec the `position` should be an `eth.Quantity`, but it's not really used as a number, just as a hash... So lets make `position` a `string` and use `common.HexToHash(position)` to parse it, just like go-ethereum does https://github.com/ethereum/go-ethereum/blob/b8a94571397c1afd5579f1e319aff05a676c917d/internal/ethapi/api.go#L727